src/components/LogIn/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { signInWithEmailAndPassword } from "firebase/auth";
+import { firebaseAuth } from "../../../firebaseConfig";
+import {
+  StyleSheet,
+  View,
+  Text,
+  TextInput,
+  Button,
+  TouchableWithoutFeedback,
+  Keyboard,
+} from "react-native";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Controller, SubmitHandler, useForm } from "react-hook-form";
+import { z } from "zod";
+import { FirebaseError } from "firebase/app";
+import getFirebaseAuthErrorMessage from "@/utils/getFirebaseAuthErrorMessage";
+
+const loginFormSchema = z.object({
+  email: z.string().email({ message: "Invalid email" }),
+  password: z
+    .string()
+    .min(6, { message: "Password must be at least 6 characters" }),
+});
+
+type LoginFormValues = z.infer<typeof loginFormSchema>;
+
+export default function LogIn() {
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+    setError,
+  } = useForm<LoginFormValues>({
+    resolver: zodResolver(loginFormSchema),
+  });
+
+  const onSubmit: SubmitHandler<LoginFormValues> = async (data) => {
+    const { email, password } = data;
+
+    try {
+      await signInWithEmailAndPassword(firebaseAuth, email, password);
+    } catch (error) {
+      if (error instanceof FirebaseError) {
+        setError("root", {
+          message: getFirebaseAuthErrorMessage(error.code),
+        });
+      }
+    }
+  };
+
+  return (
+    <TouchableWithoutFeedback onPress={Keyboard.dismiss} accessible={false}>
+      <View style={styles.container}>
+        <Text style={{ marginBottom: 15 }}>Hello! Please log in</Text>
+
+        <Controller
+          control={control}
+          name="email"
+          render={({ field: { onChange, onBlur, value } }) => (
+            <TextInput
+              autoCorrect={false}
+              style={styles.input}
+              placeholder="Email"
+              onChangeText={onChange}
+              onBlur={onBlur}
+              value={value}
+            />
+          )}
+        />
+
+        {errors.email && (
+          <Text style={{ color: "red" }}>{errors.email.message}</Text>
+        )}
+
+        <Controller
+          control={control}
+          name="password"
+          render={({ field: { onChange, onBlur, value } }) => (
+            <TextInput
+              autoCorrect={false}
```

Comment:
Make password input hidden

```suggestion
              autoCorrect={false}
              secureTextEntry={true}
```